flags/src/main/java/com/yahoo/vespa/flags/Flags.java

```diff
@@ -46,207 +42,133 @@
-            "zookeeper-server-version", "3.5.6",
-            List.of("nobody"), "2020-12-02", "2021-02-01",
-            "ZooKeeper server version, a jar file zookeeper-server-<ZOOKEEPER_SERVER_VERSION>-jar-with-dependencies.jar must exist",
-            "Takes effect on restart of Docker container",
-            NODE_TYPE, APPLICATION_ID, HOSTNAME);
-
     public static final UnboundStringFlag TLS_FOR_ZOOKEEPER_CLIENT_SERVER_COMMUNICATION = defineStringFlag(
             "tls-for-zookeeper-client-server-communication", "OFF",
-            List.of("nobody"), "2020-12-02", "2021-02-01",
+            List.of("hmusum"), "2020-12-02", "2021-02-01",
             "How to setup TLS for ZooKeeper client/server communication. Valid values are OFF, PORT_UNIFICATION, TLS_WITH_PORT_UNIFICATION, TLS_ONLY",
             "Takes effect on restart of config server",
             NODE_TYPE, HOSTNAME);
 
     public static final UnboundBooleanFlag USE_TLS_FOR_ZOOKEEPER_CLIENT = defineFeatureFlag(
             "use-tls-for-zookeeper-client", false,
-            List.of("nobody"), "2020-12-02", "2021-02-01",
+            List.of("hmusum"), "2020-12-02", "2021-02-01",
             "Whether to use TLS for ZooKeeper clients",
             "Takes effect on restart of process",
             NODE_TYPE, HOSTNAME);
 
     public static final UnboundBooleanFlag VALIDATE_ENDPOINT_CERTIFICATES = defineFeatureFlag(
             "validate-endpoint-certificates", false,
-            List.of("nobody"), "2020-12-02", "2021-02-01",
+            List.of("andreer"), "2020-12-02", "2021-02-01",
             "Whether endpoint certificates should be validated before use",
             "Takes effect on the next deployment of the application");
 
     public static final UnboundStringFlag DELETE_UNUSED_ENDPOINT_CERTIFICATES = defineStringFlag(
             "delete-unused-endpoint-certificates", "disable",
-            List.of("nobody"), "2020-12-02", "2021-02-01",
+            List.of("andreer"), "2020-12-02", "2021-02-01",
             "Whether the endpoint certificate maintainer should delete unused certificates in cameo/zk",
             "Takes effect on next scheduled run of maintainer - set to \"disable\", \"dryrun\" or \"enable\"");
 
     public static final UnboundBooleanFlag USE_ALTERNATIVE_ENDPOINT_CERTIFICATE_PROVIDER = defineFeatureFlag(
             "use-alternative-endpoint-certificate-provider", false,
-            List.of("nobody"), "2020-12-02", "2021-02-01",
+            List.of("andreer"), "2020-12-02", "2021-02-01",
             "Whether to use an alternative CA when provisioning new certificates",
             "Takes effect only on initial application deployment - not on later certificate refreshes!");
 
     public static final UnboundStringFlag DOCKER_IMAGE_REPO = defineStringFlag(
             "docker-image-repo", "",
-            List.of("nobody"), "2020-12-02", "2021-02-01",
+            List.of("valerijf"), "2020-12-02", "2021-02-01",
```

Comment:
This was marked as permanent in the speadsheet...

Reply:
Thanks for noticing, I'll move in next PR 👍